src/views/PancakeSquad/assets/ArtistBottomWave.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { Svg, SvgProps } from '@pancakeswap/uikit'
+
+const Icon: React.FC<SvgProps & { isDark: boolean }> = ({ isDark, ...props }) => {
```

Comment:
I suppose this is the curves between page sections?
We actually have `CurveDivider` component that handles it
But I'd suggest leave it like you have now, its quite a pain to do them sometimes, not worth the time 😅 
Just noting for the future 🙂